src/lib/Tests/Siteaccess/AdminSiteaccessPreviewVoterTest.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\Tests\Siteaccess;
+
+use eZ\Bundle\EzPublishCoreBundle\ApiLoader\RepositoryConfigurationProvider;
+use eZ\Publish\API\Repository\Values\Content\ContentInfo;
+use eZ\Publish\Core\MVC\ConfigResolverInterface;
+use eZ\Publish\Core\Repository\Values\Content\Location;
+use eZ\Publish\Core\Repository\Values\Content\VersionInfo;
+use EzSystems\EzPlatformAdminUi\Siteaccess\AdminSiteaccessPreviewVoter;
+use EzSystems\EzPlatformAdminUi\Siteaccess\SiteaccessPreviewVoterContext;
+use PHPUnit\Framework\TestCase;
+
+class AdminSiteaccessPreviewVoterTest extends TestCase
+{
+    public const LANGUAGE_CODE = 'eng-GB';
```

Comment:
Access level could be private or protected (depends if you designed this class as extendable)

Reply:
Done in https://github.com/ezsystems/ezplatform-admin-ui/pull/1704/commits/d3314fa6d39765a2764b1bf0e4a18eee480a3575